CHANGELOG.md

```diff
@@ -1,4 +1,72 @@
-## 2.18.0 (Unreleased)
+## 2.18.1 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.3.2355**
+- Supported Terraform version: **v0.12.x** and **v0.13.x**
+
+### Features:
+1. Implemented new resources to support TGW connect and connect peers:
```

Comment:
This is a feature, so should be TGW Connect and Connect peers. I can fix it alongside my doc PR